hive-writer/hive-writer-test.py

```diff
@@ -84,7 +83,7 @@ def old_data(start_line=0):
 
 
 if __name__ == "__main__":
-    for _ in range (5):
-        old_data(0)
-    # loop_test()
+    # for _ in range (5):
+    #     old_data(0)
+    loop_test()
```

Comment:
restore these lines to previous prior to commit,
the default shouldn't be to run the loop_test

Reply:
I was kinda thinking I was the only one running this test script so I wasn't really paying too much attention to it for others.

This script is a little dangerous in that it has the potential to dump a ton of crap into Hive very quickly! I'm probably the only person with powerful enough Hive accounts right now to do this however :-)